blog/Gemfile

```diff
@@ -0,0 +1,47 @@
+source 'https://rubygems.org'
+
+
```

Comment:
Doesn't look like you installed postgresql here. You either need to do that when you create the rails app (`rails new app_name -d postgresql) or manually include it in your Gemfile and then update your `config/database.yml` file.